pygmt/src/x2sys_cross.py

```diff
@@ -104,7 +104,7 @@ def x2sys_cross(
         via $MGD77_HOME/mgd77_paths.txt and .gmt files will be searched for via
         $GMT_SHAREDIR/mgg/gmtfile_paths).
 
-    outfile
+    outfile : None, str
```

Comment:
`outfile` already have type hints:
```suggestion
    outfile
```